### ISSUE TITLE
[RedwoodRecord] Missing types?

### ISSUE THREAD

**Issue by olance** [redwoodjs/redwood#4588]
Trying to use RedwoodRecord for the first time in a project.

I installed the package using:

```
yarn workspace api add @redwoodjs/record
```

My first model, file created by the model generator:

```js
import { RedwoodRecord } from '@redwoodjs/record'

export default class AppInstallation extends RedwoodRecord {}
```

WebStorm is showing me a warning on the module name:
<img width="422" alt="image" src="https://user-images.githubusercontent.com/314079/155840246-88d0519b-9188-4fee-bb6e-4e911ebc36cb.png">

On Visual Studio Code, the warning is slightly different, mentioning an issue with types:
<img width="1018" alt="image" src="https://user-images.githubusercontent.com/314079/155840278-dc88b5d7-ce41-4938-9a76-7f57b5c9d390.png">

There's indeed no `.d.ts` file in this module.

So in both IDEs, I'm having the following error when using my model class (`AppInstallation.create`):

```
TS2339: Property 'create' does not exist on type 'typeof AppInstallation'.
```

@cannikin are you planning on adding types to the module?
And do you know why it would be seen as not installed by WebStorm?

**Comment by cannikin** [comment]
See my reply in the announcement thread here: https://community.redwoodjs.com/t/introducing-redwoodrecord/2578/12

TL;DR There are no types right now but one person has a theory on how to add them and another may actually try to implement it.

**Comment by olance** [comment]
Thanks for the link @cannikin :)

I guess I'll leave with the Webstorm issue, I guess it's probably not Redwood-specific.

Will be waiting for the TS types patiently 🙂
